.flake8

```diff
@@ -14,7 +14,7 @@ ignore =
     # these ignores are from flake8-bugbear; please fix!
     B007,B008,B017,B019,B020,B023,B024,B026,B027,B028,B903,B904,B905,B906,B907
     # these ignores are from flake8-comprehensions; please fix!
-    C407
+    C407,C419,
```

Comment:
Are there blcokers for resolving C419 in this PR? Looks like it should just be removing the brackets of any/all([..]) calls but I could be missing something.

Reply:
The main block is last I checked `@torch.jit.script` did not have generator support, but I haven't look into that since torch `1.x` maybe torch.jit.script can handle generators now? This means that a workaround to support `any/all` function calls is to use list comprehensions instead which are supported (even if they do not shortcircuit). We could blacklist the check only in parts of the codebase that need to support `torch.jit.script`, but I am not sure which parts fo the code base that would be. Would banning the check in `torch.nn.modules` be sufficient?